content/solutions/pipeline.adoc

```diff
@@ -29,8 +29,7 @@ features like:
 * checking the pipeline definition into source control (`Jenkinsfile`)
 * support for extending the domain specific language with additional,
   organization specific steps, via the
-  "link:https://www.cloudbees.com/blog/jenkins-workflow-using-global-library-implement-re-usable-function-call-secured-http-endpoint[global
-  library]" feature
+  "link:https://github.com/jenkinsci/workflow-cps-global-lib-plugin[global library]" plugin
```

Comment:
Sorry, for the piecemeal review but I think you should like to this page instead: 
https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Shared+Groovy+Libraries+Plugin

Or even better call them by the new doc page here: 
https://jenkins.io/doc/book/pipeline/shared-libraries/

And the text can be "shared library feature".
😄